GravatarApp/Welcome/WelcomeViewModel.swift

```diff
@@ -94,8 +94,10 @@ class WelcomeViewModel: ObservableObject {
     }
 
     func logout() async {
-        guard let profile = profileResult?.value() else { return }
```

Comment:
This caused a bug because the profile can't be fetched if the token is expired so the logout and deletion of token failed. Updated this to use the persisted user id.